invenio_theme_tugraz/templates/invenio_theme_tugraz/navbar.html

```diff
@@ -62,11 +62,11 @@
         </div>
         <div class="affiliation-text">
           <a title="RDM" href="{{url_for('invenio_theme_tugraz.index')}}">
-            TU Graz
+            TU GRAZ
             <br>
-            {{_ ("Repository")}}
+            {{_ ("REPOSITORY")}}
             <br>
-            {{_ ("Library & Archives")}}
+            {{_ ("LIBRARAY & ARCHIVES")}}
```

Comment:
It is difficult here what type this is. It is not a refactor type for me, but it is also not really of the type feature. Maybe we should add a new type like "modification"?

Reply:
sounds good.